cmd/log.go

```diff
@@ -85,8 +106,19 @@ func Log(c *git.Client, args []string) error {
 	if err != nil {
 		return err
 	}
-	return git.RevListCallback(c, git.RevListOptions{Quiet: true}, []git.Commitish{commit}, nil, func(s git.Sha1) error {
+
+	var numCommits = 0
+
+	err = git.RevListCallback(c, git.RevListOptions{Quiet: true}, []git.Commitish{commit}, nil, func(s git.Sha1) error {
```

Comment:
I think the max-count is supposed to be a parameter to rev-list that log just passes down.. can we move this into RevListCallback?

Reply:
Thanks, yes it looks like the pretty printing should be pushed down there too. It looks like the printing of the output will be the only thing that log command will do in the future.